content/blog/2018/07/2018-07-03-remoting-over-message-bus-alpha-release.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "GSoC Project Update: Alpha release of remoting-kafka-plugin"
+tags:
+- plugins
+- gsoc
+- gsoc2018
+- remoting
+- alpha-release
+author: pvtuan10
+---
+
+=== Overview
+https://github.com/jenkinsci/remoting-kafka-plugin[Remoting Kafka Plugin] is a plugin developed under https://jenkins.io/projects/gsoc/[Jenkins Google Summer of Code 2018]. The plugin allows users to connect agents to Jenkins master using Kafka to solve the problem of current direct TCP communication between master and agent. A quick introduction of the project could be found in this  https://jenkins.io/blog/2018/06/18/remoting-over-message-bus/[introduction blogpost].
+
+We are currently in the second phase of the project and I am happy to announce that we have recently released alpha version of the plugin under https://jenkins.io/doc/developer/publishing/releasing-experimental-updates/#configuring-jenkins-to-use-experimental-update-center[Experimental Update Center]. You can check the https://github.com/jenkinsci/remoting-kafka-plugin/blob/master/CHANGELOG.md[CHANGELOG] to see the features developed in this initial release.
+
+=== How to use the plugin?
+The instructions to run the plugin in alpha version are written https://github.com/jenkinsci/remoting-kafka-plugin#how-to-use-the-plugin-in-alpha-version[here]. Feel free to have a try and let us know your feedback by https://gitter.im/jenkinsci/remoting[Gitter] or the https://groups.google.com/forum/?nomobile=true#!forum/jenkinsci-dev[mailing list].
+
+=== Links
+
+* https://github.com/jenkinsci/remoting-kafka-plugin/blob/master/CHANGELOG.md#100-alpha-1[Alpha Changelog]
+* https://jenkins.io/blog/2018/06/18/remoting-over-message-bus/[Introduction Blogpost]
```

Comment:
should be just "/blog/2018/06/18/remoting-over-message-bus/" so that you can test links in the local run

Reply:
Relative link doesn't work

Reply:
@pvtuan10  How does it not work when run locally using `make run`?

Reply:
I got this when I tried to run locally
![screenshot from 2018-07-04 11-49-22](https://user-images.githubusercontent.com/13913977/42257323-537b34e0-7f80-11e8-8d4b-95d18eb0cf90.png)

Reply:
Try `link:/blog/2018/06/18/remoting-over-message-bus/[Introduction Blogpost]`. Likely you we trying relative links without `link:` prefix